src/actionlist/action-list-item.scss

```diff
@@ -35,9 +39,11 @@
 
   // only hover li without list as children
   &:not(.ActionList-item--hasSubItem) {
-    &:hover {
-      cursor: pointer;
-      background-color: var(--color-action-list-item-default-hover-bg);
+    @media (hover: hover) {
```

Comment:
I haven't tested this recently, but a while back using `@media (hover: hover)` was not really reliable for "hybrid devices" (like a laptop with a touch screen, or connecting a mouse to an iPad). The `@media (hover: hover)` would only work for what the browser thought is the "primary" input but would not adjust when switching touch/mouse on the fly. Maybe that improved by now and it is a bit of an edge case, so maybe still fine to keep it until we hear some reports about it. 🤷

Reply:
@simurai that's a great point! I think this is still an expected behavior for hybrid devices, according to the latest [W3C Media Query specs](https://drafts.csswg.org/mediaqueries/#hover).

> Example 44
> For example, on a touch screen device that can also be controlled by an optional mouse, the hover media feature should match hover: none, as the primary pointing device (the touch screen) does not allow the user to hover.
However, despite this, the optional mouse does allow users to hover. Authors should therefore be careful not to assume that the ':hover' pseudo class will never match on a device where 'hover:none' is true, **but they should design layouts that do not depend on hovering to be fully usable**.

I think the current solution is a good compromise. Showing or not showing a hover state (in case of a laptop with touch screen, or a touch device with a plugged-in mouse, respectively) is not an unexpected behavior for these non-primary types of interactions, I'd say.

👍